app/code/AKisilenko/ModuleLesson6/Cron/Status.php

```diff
@@ -22,38 +30,67 @@ class Status
     /**
      * Status constructor.
      * @param LoggerInterface $logger
+     * @param ScopeConfigInterface $scopeConfig
      * @param CollectionFactory $askQuestionsFactory
      */
     public function __construct(
         LoggerInterface $logger,
+        ScopeConfigInterface $scopeConfig,
+//        Config $config,
         CollectionFactory $askQuestionsFactory
     ) {
+//        $this->dateQuantity = $config;
         $this->logger = $logger;
+        $this->scopeConfig = $scopeConfig;
         $this->askFactory = $askQuestionsFactory;
     }
 
     /**
+     * @param string $key
+     * @param string $section
+     * @return mixed
+     */
+    private function isEnabled($key = 'enable_cron', $section = 'cron')
```

Comment:
I think it's not a good idea to pass the section and key into the method isEnabled, If somebody can replace the key and section identifiers with some other values, so isEnabled will not return the module's enabled status but will return something else, so the method's name is incorrect. So it will be the place where other developers will do something incorrect, passing some strange input data.
IsEnabled shouldn't receive any input arguments. 
You can do this in some private method like getAskQuestionConfigValue or so, but I dont think it's correct. Better is to make the code which other developers cannot run in incorrect way.
So my adive is to define the constants for each section, something like 
const XML_PATH_ASK_QUESION_CRON_IS_ENABLED =  'ask_question_options/cron/enable_cron';
const XML_PATH_ASK_QUESION_CRON_FREQUENCY =  'ask_question_options/cron/frequency';
and use this constants in each method. So nobody could pass the incorrect key.
Please refer Magento core how they do it

Reply:
please check if I understand you correctly